cmake/thirdparty/BLTSetupCUDA.cmake

```diff
@@ -67,18 +50,6 @@ if (NOT BLT_ENABLE_CLANG_CUDA)
     enable_language(CUDA)
 endif()
 
-if(CMAKE_CUDA_STANDARD STREQUAL "17")
```

Comment:
I just moved these checks to after the call to find_package(CUDA) so that we can use CUDA_VERSION_STRING.

Reply:
We really need to move away from `find_package(CUDA)` because it is deprecated since 3.10 and also in 4.0 if i remember correctly throws a error or a loud warning.

https://cmake.org/cmake/help/latest/module/FindCUDA.html